### DISCUSSION TITLE
dynamically insert a table row using inertia

### DISCUSSION THREAD

**Discussion by eaktadiur** [inertiajs/inertia#1546] [Help (Vue)]
### Version:

- `@inertiajs/vue3` version: #.#.#

### Describe the problem:
I am using Laravel 10
This is my table with five array object

<button class="btn btn-primary" @click="findAndaddUser(6)">Add Prod</button>

<table class="table table-bordered table-hover">
    <thead>
        <tr>
            <th>S/N</th>
            <th>Name</th>
            <th>Title</th>
        </tr>
    </thead>
    <tbody>
        <tr v-for="(row, i) in users" :key="row.id">
            <td>{{ i + 1 }}</td>
            <td>{{ row.name }}</td>
            <td>{{ row.title }}</td>
        </tr>
    </tbody>
</table>

when I will click the button it will call the findAndaddUser function which will return an object with "{id: 6, name:'test 6', title: 'account' }"

I need to push this response to users array

**Comment by craigrileyuk** [comment]
Dude, seriously?

This is for reporting **issues** with the codebase, not a place for asking for help with your coding projects.
